mephisto/utils/scripts.py

```diff
@@ -19,154 +20,83 @@
+    return LocalMephistoDB(database_path=database_path)
 
-    def add_datapath_arg(self, argument_group=None):
-        """
-        Add datapath for use with mephisto
-        """
-        if argument_group is None:
-            argument_group = self
-        argument_group.add_argument(
-            "-dpath", "--datapath", default=None, help="Data path for this mephisto run"
-        )
 
-    def parse_launch_arguments(
-        self, args=None
-    ) -> Tuple[str, str, "MephistoDB", Dict[str, Any]]:
-        """
-        Parse common arguments out from the command line, returns a 
-        tuple of the architect type, the requester name to use, the
-        MephistoDB to run with, and any  additional arguments parsed 
-        out by the argument parser
-
-        Defaults to a mock architect with a mock requester with no arguments
-        """
-        args, _unknown = self.parse_known_args(args=args)
-        arg_dict = vars(args)
-        requester_name = arg_dict["requester_name"]
-        provider_type = arg_dict["provider_type"]
-        architect_type = arg_dict["architect_type"]
-        datapath = arg_dict["datapath"]
-
-        if datapath is None:
-            datapath = get_root_data_dir()
-
-        database_path = os.path.join(datapath, "database.db")
-        db = LocalMephistoDB(database_path=database_path)
-
-        if requester_name is None:
-            if provider_type is None:
-                print("No requester specified, defaulting to mock")
-                provider_type = "mock"
-            if provider_type == "mock":
-                req = get_mock_requester(db)
-                requester_name = req.requester_name
-            else:
-                reqs = db.find_requesters(provider_type=provider_type)
-                # TODO (#93) proper logging
-                if len(reqs) == 0:
-                    print(
-                        f"No requesters found for provider type {provider_type}, please "
-                        f"register one. You can register with `mephisto register {provider_type}`, "
-                        f"or `python mephisto/client/cli.py register {provider_type}` if you haven't "
-                        "installed Mephisto using poetry."
-                    )
-                    exit(1)
-                elif len(reqs) == 1:
-                    req = reqs[0]
-                    requester_name = req.requester_name
-                    print(
-                        f"Found one `{provider_type}` requester to launch with: {requester_name}"
-                    )
-                else:
-                    req = reqs[-1]
-                    requester_name = req.requester_name
-                    print(
-                        f"Found many `{provider_type}` requesters to launch with, "
-                        f"choosing the most recent: {requester_name}"
-                    )
+def augment_config_from_db(db: "MephistoDB", script_cfg: DictConfig) -> DictConfig:
```

Comment:
this is very nit-picky of me but I sort of like the argument signature to be reversed such as (script_cfg, db) so it's parallel to the name of the fn `augment_config_from_db`.